lib/src/components/NxTag/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {ReactNode, WeakValidationMap} from 'react';
+import * as PropTypes from 'prop-types';
+
+export interface Props {
+  onTagSelect?: (() => void) | null;
+  tagSelected?: boolean;
+  children: ReactNode;
+  tagColor?: string | null;
```

Comment:
Some suggestions to make the prop names clearer and less redundant:
```suggestion
  onSelect?: (() => void) | null;
  isSelected?: boolean;
  children: ReactNode;
  color?: string | null;
```

Reply:
> Some suggestions to make the prop names clearer and less redundant:

Even though it looks like this change was accepted, I don't see it in the current state of the diff.  Maybe it got overwritten somehow.

Reply:
> > Some suggestions to make the prop names clearer and less redundant:
> 
> Even though it looks like this change was accepted, I don't see it in the current state of the diff. Maybe it got overwritten somehow.

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/f4f84d6759151742864177ddfd3fe41e574e7758

Reply:
:+1: seems like you forgot to update the examples though; I'm getting build errors in them

Reply:
> 👍 seems like you forgot to update the examples though; I'm getting build errors in them

Yeah I noticed, fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/f4f84d6759151742864177ddfd3fe41e574e7758